redfish-core/lib/memory.hpp

```diff
@@ -33,17 +33,12 @@
 #include <sdbusplus/unpack_properties.hpp>
 
 #include <array>
+#include <functional>
```

Comment:
Additionally, several more includes

```
#include <boost/urls/format>

#include <cstdint>
#include <memory>
#include <string>
#include <vector>
```

Reply:
The boost one is already there on line 30 (although it is using .hpp extension, should be the same thing correct?)

I'll add the other ones.